src/modules/alwaysontop/AlwaysOnTop/AlwaysOnTop.h

```diff
@@ -47,6 +47,9 @@ class AlwaysOnTop : public SettingsObserver
     HWND m_window{ nullptr };
     HINSTANCE m_hinstance;
     std::map<HWND, std::unique_ptr<WindowBorder>> m_topmostWindows{};
+    HANDLE m_hPinEvent;
+    std::thread m_thread;
+    bool m_powerToysPid;
```

Comment:
Let's rename it since it doesn't store pid, only a flag if we use llkh.
And it won't be changed later, so const.

```suggestion
    const bool m_useCentralizedLLKH;
```